packages/js/src/ui/components/InboxTabs/InboxTabs.tsx

```diff
@@ -43,7 +44,19 @@ export const InboxTabs = (props: InboxTabsProps) => {
       ...tab,
       rightIcon:
         tab.label === activeTab() ? (
-          <Check class={style('moreTabs__dropdownItemRight__icon', 'nt-size-3')} />
+          <IconRendererWrapper
+            iconKey="check"
+            class={style('moreTabs__dropdownItemRight__icon', 'nt-size-3', {
+              iconKey: 'check',
+            })}
+            fallback={
+              <DefaultCheck
+                class={style('moreTabs__dropdownItemRight__icon', 'nt-size-3', {
+                  iconKey: 'check',
+                })}
```

Comment:
Not huge fan of the dulication of class names, but to incapsulate this inside the icon renderer wrapper to reuse the class name resulted in quite a weird wrapper and syntax